hbase-replication/src/main/java/org/apache/hadoop/hbase/replication/FSReplicationPeerStorage.java

```diff
@@ -227,6 +227,16 @@ public ReplicationPeerConfig getPeerConfig(String peerId) throws ReplicationExce
     }
   }
 
+  @Override
+  public long getPeerCreateTime(String peerId) throws ReplicationException {
+    Path peerDir = getPeerDir(peerId);
+    try {
+      return fs.getFileStatus(peerDir).getModificationTime();
+    } catch (IOException e) {
```

Comment:
We need to check null or catch FileNotFoundException here in case the peer does not exist?

Reply:
And it should be ctime instead of mtime?

Reply:
> And it should be ctime instead of mtime?

First, I try to use FileStatus.getCreationTime(), but the code shown in FileStatus class without the method.
I checked that the version of the dependent hadoop-client is 3.3.5
Do you have time to look at this question?

Reply:
Checked the code, there is no ctime for hadoop...

So for FS based storage, I think we need a separated file to store the creation time...

Reply:
it's ok